src/Analyzers/Microsoft.Extensions.ExtraAnalyzers/Common/CallAnalysis/Fixers/LegacyLoggingFixer.cs

```diff
@@ -367,16 +367,21 @@ private static string GetNamespace(ClassDeclarationSyntax cl)
             loggerParam
         };
 
-        if (details.ExceptionParamIndex >= 0)
+        if (details.ExceptionParamIndex >= 0 && invocationOp.Arguments[details.ExceptionParamIndex].Value.Type is ITypeSymbol symbol)
         {
-            parameters.Add(gen.ParameterDeclaration("exception", gen.TypeExpression(invocationOp.Arguments[details.ExceptionParamIndex].Value.Type)));
+            parameters.Add(gen.ParameterDeclaration("exception", gen.TypeExpression(symbol)));
         }
 
         var index = 0;
         if (details.InterpolationArgs != null)
         {
             foreach (var o in details.InterpolationArgs)
             {
+                if (o.Type is null)
```

Comment:
What's this about?

Reply:
Newer packages have nullable annotations which trigger build warnings.